utils/netlist-checks/power-states.stanza

```diff
@@ -0,0 +1,51 @@
+#use-added-syntax(jitx)
+defpackage ocdb/utils/netlist-checks/power-states :
+  import core
+  import collections
+  import jitx
+  import jitx/commands
+  import lang-utils
+  import ocdb/utils/netlist-checks/utils
+  
+
+public defn check-power-states (module:JITXObject, connections:Connections) :
```

Comment:
TODO: this needs to be updated as well, but it's not a pressing priority at the moment.